Tests/MongoSwiftTests/ConnectionStringTests.swift

```diff
@@ -110,17 +110,25 @@ let shouldWarnButLibmongocErrors: [String: [String]] = [
 // tests we skip because we don't support the specified behavior.
 let skipUnsupported: [String: [String]] = [
     // we don't support maxIdleTimeMS.
-    "connection-pool-options.json": [
-        "Too low maxIdleTimeMS causes a warning",
-        "Non-numeric maxIdleTimeMS causes a warning",
-        "Valid connection pool options are parsed correctly"
-    ],
+    "connection-pool-options.json": ["*"],
     // requires maxIdleTimeMS
-    "connection-options.json": [
-        "Valid connection and timeout options are parsed correctly"
-    ],
-    "compression-options.json": ["Multiple compressors are parsed correctly"], // requires Snappy, see SWIFT-894
-    "valid-db-with-dotted-name.json": ["*"] // libmongoc doesn't allow db names in dotted form in the URI
+    "connection-options.json": ["*"],
+    "compression-options.json": ["*"], // requires Snappy, see SWIFT-894
+    "valid-db-with-dotted-name.json": ["*"], // libmongoc doesn't allow db names in dotted form in the URI
+
+    // Disabled for MongoConnectionString
+    "invalid-uris.json": ["*"],
```

Comment:
There are some valid test cases in this file actually, instead of `"*"` could we specify only the tests that verify options / auth / other unsupported behavior? Feel free to update `shouldSkip` to check for substring equality instead of exact equality if you think it'd make defining this skip list a little easier. (e.g. `"invalid-uris.json": [ "option", "username", "password"]`)

Reply:
done